WordPress/Sniffs/WP/GetMetaSingleSniff.php

```diff
@@ -33,6 +33,48 @@ final class GetMetaSingleSniff extends AbstractFunctionParameterSniff {
 	 */
 	const METRIC_NAME = 'get_*meta() function called with $single parameter';
 
+	/**
```

Comment:
Can the order of these constants please be reversed ?

To me, the logical order would be to go from generic to more specific.
And even if that would be too arbitrary, we could use alphabetical order and that would still result in the same: the `GENERIC_META_FUNCTIONS_FORMAT` first, the `SPECIFIC_META_FUNCTIONS_FORMAT` constant second.